Compile error for Windows

```
Error 30 error C2440: 'return' : cannot convert from 'std::string' to 'char *'  C:\Users\Harvir\Desktop\Astron\astron\src\util\password_prompt.cpp 21 1 astrond
33 IntelliSense: no suitable conversion function from "std::string" to "char *" exists  c:\Users\Harvir\Desktop\Astron\astron\src\util\password_prompt.cpp 21 12 astrond
```

Branch: master
Compiler: VS2012
OS: Windows ¬.¬
I should get into the habit of testing PRs...

--- a/src/util/password_prompt.cpp
+++ b/src/util/password_prompt.cpp
@@ -18,7 +18,7 @@ char *getpass(const char *prompt)
 
     string password;
     getline(cin, password);
-    return password;
+    return password.c_str();
 }
 
 

